app/matches.py

```diff
@@ -64,26 +73,18 @@ def getMatchesWithPlayer(id):
           to_return.append(match)
     return json.dumps([m.serialize() for m in to_return])
 
+
   def createMatch(event, playersInMatch, score, category):
     if ((event is None) | (playersInMatch[0] is None) | (playersInMatch[1] is None) | (score is None)):
```

Comment:
```suggestion
    if ((event is None) | (playersInMatch is None) | (score is None)):
```
We can make better use of the nice new methods you just created!